### PR TITLE
adding PRs specific to the RC2 release

### PR DESCRIPTION
**Pull request by esc**
As title

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -18,7 +18,10 @@ Pull-Requests:
 * PR `#726 <https://github.com/numba/llvmlite/pull/726>`_: Disallow alwaysinline and noinline on functions (`Graham Markall <https://github.com/gmarkall>`_)
 * PR `#730 <https://github.com/numba/llvmlite/pull/730>`_: Don't export LLVM symbols when linking statically (`Chris Burr <https://github.com/chrisburr>`_ `esc <https://github.com/esc>`_)
 * PR `#732 <https://github.com/numba/llvmlite/pull/732>`_: Docs/remove travis (`esc <https://github.com/esc>`_)
-* PR `#734 <https://github.com/numba/llvmlite/pull/734>`_: bump llvmdev to 11 (`esc <https://github.com/esc>`_)
```
Comment (stuartarchibald) [inline CHANGE_LOG:21]: Is this deletion intentional? #734 is on the release branch https://github.com/numba/llvmlite/commits/release0.37 ?

Comment (sklam) [reply CHANGE_LOG:21]: looks accidental to me

File: CHANGE_LOG [hunk 2]
```diff
@@ -18,7 +18,10 @@ Pull-Requests:
...
+* PR `#736 <https://github.com/numba/llvmlite/pull/736>`_: manylinux llvmdev upgrade to LLVM 11 / manylinux2010 to manylinux2014 upgrade (`esc <https://github.com/esc>`_)
```
Comment (stuartarchibald) [inline CHANGE_LOG:21]:
```suggestion
* PR `#734 <https://github.com/numba/llvmlite/pull/734>`_: bump llvmdev to 11 (`esc <https://github.com/esc>`_)
* PR `#736 <https://github.com/numba/llvmlite/pull/736>`_: manylinux llvmdev upgrade to LLVM 11 / manylinux2010 to manylinux2014 upgrade (`esc <https://github.com/esc>`_)
```

**Review by stuartarchibald — APPROVED** [review]
Looks good, thanks!

File: CHANGE_LOG (hunk 1 above)
Comment (esc) [reply CHANGE_LOG:21]: yes, `d` key stuck in vim.

**Pull request merged**